### PR TITLE
Added Accordion

### LINKED ISSUE #2: Adding Accordion Component Folder in the ui library
Adding a folder for Accordion Component

### PR DESCRIPTION
**Pull request by Palaksharma23**
Fixes #2
I have added Accordion component in this pull request

### CODE REVIEW

File: src/components/Accordion/AccordionItem.vue [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  color: v-bind(headingcolor);
+  padding-right: 1%;
+}
+
+.active .label_secondary::before {
+  content: "-";
+  float: right;
+  color: v-bind(headingcolor);
+  padding-right: 1%;
+}
```
Comment (Janvi-Thakkar) [inline src/components/Accordion/AccordionItem.vue:104]: Good work @Palaksharma23 💯.
I am providing this as an example and you can change it at other places accordingly. as float, color, and padding properties are not changing for active classes as well, you don't need to repeat them again. you just need to mention **content:'-'** inside **.active .label_secondary:before**. rest will be inherited from **.label_secondary:before**

File: src/components/Accordion/AccordionItem.vue [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      type: String,
+      default: "1.4vw",
+    },
+    fsheading: {
+      type: String,
+      default: "2vw",
+    },
+    Bgheading: {
+      type: String,
+      default: "#2e382b",
```
Comment (Janvi-Thakkar) [inline src/components/Accordion/AccordionItem.vue:28]: Either give the name of variables in camelCase or PascalCase or snake_case.

Comment (Palaksharma23) [reply src/components/Accordion/AccordionItem.vue:28]: let's go with camelCase

**Review by Janvi-Thakkar — COMMENTED** [review]
giving me errors
image

**Comment by YashKandalkar** [comment]
@Palaksharma23 I think you have a wrong onclick event handler somewhere, it throws an error when the description is clicked. 
Please have a look into it.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**